sales.html

```diff
@@ -17,6 +17,34 @@ <h1><img src="images/salmon.png" class= "salmonPic" rel= "Salmon Picture">  Pat'
     <h2 id = "salesData">Cookie Sales by Store Location</h2>
 
     <table id="dailyStoreSales"></table>
+
+    <br />
+  
+    <form id="newStore">
+      <fieldset>
+        <legend>Add a New Store Location</legend>
+        <br />
+  
+        <label for="newStoreName">Store Name: </label>
+        <input name="newStoreName" type="text">
+        <br />
+  
+        <label for="newMin">Hourly Min. Customers: </label>
+        <input name="newMin" type="text">
```

Comment:
Since these should be number values, you can use `type="number"` instead to ensure the user enters a number.